apps/ledger-live-mobile/e2e/specs/nftGallery.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import { loadConfig } from "../bridge/server";
+import { tapByElement } from "../helpers";
+import WalletTabNavigatorPage from "../models/wallet/walletTabNavigator";
+import NftViewerPage from "../models/nft/nftViewerPage";
+import NftGalleryPage from "../models/wallet/nftGalleryPage";
+
+let portfolioPage: PortfolioPage;
+let walletTabNavigatorPage: WalletTabNavigatorPage;
+let nftGalleryPage: NftGalleryPage;
+let nftViewerPage: NftViewerPage;
+
+describe("NFT Gallery screen", () => {
+  beforeAll(async () => {
+    await loadConfig("2_accounts_eth_with_tokens_and_nft", true);
+    portfolioPage = new PortfolioPage();
+    walletTabNavigatorPage = new WalletTabNavigatorPage();
+    nftGalleryPage = new NftGalleryPage();
+    nftViewerPage = new NftViewerPage();
+  });
+
+  it("should open on Portofolio page", async () => {
+    await expect(portfolioPage.getSettingsButton()).toBeVisible();
+  });
+
+  it("should see NFT tab", async () => {
+    await expect(walletTabNavigatorPage.getNftGalleryTab()).toBeVisible();
+  });
+
+  it("should navigate to NFT gallery on NFT tab press", async () => {
+    await tapByElement(walletTabNavigatorPage.getNftGalleryTab());
+    await expect(nftGalleryPage.getNftList()).toBeVisible();
+  });
+
+  it("should navigate to NFT viewer page on NFT gallery item press", async () => {
+    await tapByElement(nftGalleryPage.getNftListItems());
```

Comment:
Similar here. Make a `nftGalleryPage.navigateToFirstNftItem()` in the `apps/ledger-live-mobile/e2e/models/wallet/nftGalleryPage.ts` file

Reply:
Done